.idea/libraries/Dart_SDK.xml

```diff
@@ -1,26 +1,26 @@
 <component name="libraryTable">
   <library name="Dart SDK">
     <CLASSES>
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/async" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/cli" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/collection" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/convert" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/core" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/developer" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/ffi" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/html" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/indexed_db" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/io" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/isolate" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/js" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/js_util" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/math" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/mirrors" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/svg" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/typed_data" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/web_audio" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/web_gl" />
-      <root url="file://$PROJECT_DIR$/../flutter/bin/cache/dart-sdk/lib/web_sql" />
+      <root url="file://C:/src/flutter/bin/cache/dart-sdk/lib/async" />
```

Comment:
Should this file be ignored? Because I am using a mac and my path is different, I guess you are using a Windows machine.

Reply:
Yes I just updated it.